sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/WhereParser.java

```diff
@@ -0,0 +1,203 @@
+package io.stargate.sgv2.restsvc.resources;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.StreamReadFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.stargate.grpc.Values;
+import io.stargate.proto.QueryOuterClass;
+import io.stargate.proto.Schema;
+import io.stargate.sgv2.common.cql.builder.BuiltCondition;
+import io.stargate.sgv2.common.cql.builder.Predicate;
+import io.stargate.sgv2.restsvc.grpc.ToProtoConverter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Helper class that handles parsing of "WHERE" clause for GET rows endpoint.
+ *
+ * <p>Note: code modified from StargateV1 "WhereParser" ({@code
```

Comment:
Wondering if this comment will make sense in the future after SGv2 is the official version - won't the class you're referencing be removed?

Reply:
Yes it will. Not sure how to balance short- and long-term needs here.